expansion/src/main/java/no/unit/nva/expansion/model/ExpandedDatabaseEntry.java

```diff
@@ -26,4 +28,11 @@ default String toJsonString() {
             throw new RuntimeException(e);
         }
     }
+
+    SortableIdentifier fetchIdentifier();
+
+    default URI fetchId() {
+        return new UriWrapper(ID_NAMESPACE).addChild(this.fetchIdentifier().toString()).getUri();
+    }
```

Comment:
methods for acquiring the identifier and id values without interfering with the serialization.